tests/python/driver/tvmc/test_frontends.py

```diff
@@ -269,6 +269,10 @@ def test_load_quantized_model__pth(pytorch_mobilenetv2_quantized):
         assert p.dtype in ["int8", "uint8", "int32"]  # int32 for bias
 
 
+@pytest.mark.skipif(
+    platform.machine() == "aarch64",
+    reason="Currently failing on AArch64 - see https://github.com/apache/tvm/issues/10673",
```

Comment:
this test isnt explicitly mentioned in this issue, can you add it?

Reply:
Done now. thanks